migrations_scripts/1_use_registry_and_add_adapters.js

```diff
@@ -401,6 +407,19 @@ module.exports = async (deployer, network, accounts) => {
     'protocol-icons.s3.amazonaws.com/0x-staking.png',
     '0',
   ]);
+  await deployer.deploy(IdleAdapter, { from: accounts[0] })
+    .then(() => {
+      adapters[13].push(IdleAdapter.address);
+      tokens[13].push(idleAdapterTokens);
+    });
+  protocolNames.push('idle.finance');
+  metadata.push([
+    'idle.finance',
```

Comment:
this field is protocol name, may be it's better to leave 'Idle' here?

Reply:
ok